execute_pipeline.py

```diff
@@ -82,6 +106,18 @@ def _CreateArgumentParser():
       type=str,
       default="..",
       help="Root directory where the input, output, and tool repositories live")
+  parser.add_argument(
+      "--local_repo",
+      type=str,
+      default=None,
+      help="Directory where local proto and gapic configs lives. This flag is "
+           "used for remote execution only.")
+  parser.add_argument(
+        "--env",
+        type=str,
+        default=None,
+        help="Specify which environment should be used to execute pipeline. "
+             "Valid values are 'LOCAL' and 'REMOTE'")
```

Comment:
Thanks for catching this. It is intended to be case-insensitive. Changed line 60 and 172 to fix the bug.